init-openstack.html.md.erb

```diff
@@ -151,28 +151,36 @@ You must create and configure two Security Groups to restrict incoming network t
     $ git clone https://github.com/cloudfoundry/bosh-deployment
 
     # Fill below variables (replace example values) and deploy the Director
+    $ DIRECTOR_NAME='bosh-1'     # arbitrary name for the director
```

Comment:
I wanted to do that initially. However, when breaking long lines with `\` one cannot add comments afterwards. Also, I wanted to keep the code sample c&p-able.

One way I see is e.g.:
```
command \
    # a comment on arg \
    -some arg \
    # a comment on other_arg \
    -some other_arg
```

Just let me know what you'd prefer and I can adopt my change.